charts/opentelemetry-demo/templates/component.yaml

```
@@ -12,6 +12,7 @@
         {{- include "otel-demo.deployment" $config -}}
         {{- include "otel-demo.service" $config -}}
         {{- include "otel-demo.ingress" $config -}}
+        {{- include "otel.demo.configmap" $config -}}
```

use otel-demo instead of otel.demo for the name here to stay consistent with other names.

```suggestion
        {{- include "otel-demo.configmap" $config -}}
```